base/OpenGLVisitor.cpp

```diff
@@ -185,32 +182,29 @@ void OpenGLVisitor::visit(Group* g) {
             vector<int>::iterator indexIter = (*polyIter)->m_vIndex.begin();
             vector<int>::iterator indexEnd = (*polyIter)->m_vIndex.end();
             #ifdef HAVE_OPENGLES
-            GLuint vbo;
-            glGenBuffers(1, &vbo);
-            glBindBuffer(GL_ARRAY_BUFFER, vbo);
-            float *verts = new float[(*polyIter)->m_vIndex.size() * 5];
-            glTexCoordPointer(2, GL_FLOAT, 5 * sizeof(float), NULL);
-            glVertexPointer(3, GL_FLOAT, 5 * sizeof(float), (const float *)NULL + 2);
+            float *texcoord_pointer = new float[(*polyIter)->m_vIndex.size() * 2];
```

Comment:
would it make sense to have this array allocated once ? like in a userdata member of poly class , or maybe an other design pattern... I need to look back into the architecture ...